### ISSUE TITLE
Adding installation help

### ISSUE THREAD

**Issue by keli5** [Litleck/Minkinator#1]
Hello! `canvas` and `sqlite3` can be notoriously difficult packages to install sometimes with NPM. It would likely be helpful to add help for common issues such as ELIFECYCLE errors, or at least links to some helpful resources, such as a StackOverflow question. I've personally been struggling for a bit to get canvas to cooperate, and haven't found anything useful. At the least, it would probably help to include some quick fixes that can get the bot working until the user figures out how to get the actual version of the package working.
Thanks!

**Comment by Litleck** [comment]
I'll work on it, canvas and sqlite3 have caused me problems several times but I'll make a note to sometime do this.
